manifests/e/EclipseFoundation/AdoptiumOpenJDK/17/17.0.0.35/EclipseFoundation.AdoptiumOpenJDK.17.installer.yaml

```
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: EclipseFoundation.AdoptiumOpenJDK.17
+PackageVersion: 17.0.0.35
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17%2B35/OpenJDK17-jdk_x64_windows_hotspot_17_35.msi
+  InstallerSha256: 4A0350A9E3E0CF3BBA9A57890BB0B223B7C47883169730F122100FC31D61D486
+  ProductCode: "{8F6755A2-8D29-4AFF-B437-337718361BD1}"
```

Where did you get this ProductCode from?
```suggestion
  ProductCode: "{D669C4CD-053E-4C32-853C-9FC53FA35FED}"
```

---

Copied over from the previous version 16. Where did you get this one from?

---

There are multiple ways of getting the ProductCode, the simplest is probably by checking the Uninstall Registry path (RevoUninstall will bring you there)
You can also use wingetcreate to update/create a new Package

FYI: The ProductCode is different for each MSI/MSIX package, exe installers have a static ProductCode that looks different